druid/src/win_handler.rs

```diff
@@ -80,6 +80,8 @@ struct DialogInfo {
     id: WindowId,
     /// The command to send if the dialog is accepted.
     accept_cmd: Selector<FileInfo>,
+    /// The command to send if the dialog is accepted with multiple files.
+    accept_multiple_cmd: Option<Selector<Vec<FileInfo>>>,
```

Comment:
Is it necessary for this to be an `Option`? I mean, I see that it's not used for save dialogs, but then save dialogs shouldn't get configured for multi-selection anyway...

Reply:
It wasn't an `Option` in my first implementation, I made it so to avoid having the field in save dialogs as you noticed.
If you think this isn't a problem though, having it always set makes the code nicer elsewhere, so I'm going to change this.